src/dataHelpers.jl

```diff
@@ -135,3 +135,29 @@ function importDepletion(dataType)
     df[df[:, :Background] .== "gcKO", [:FcgRI, :FcgRIIB, :FcgRIII, :FcgRIV]] .= 0.0
     return df
 end
+
+function importAlterMSG()
+    dataDir = joinpath(dirname(pathof(FcgR)), "..", "data", "alter-MSB")
+    
+    dfF = CSV.read(joinpath(dataDir, "data-function.csv"))
+    dfGP = CSV.read(joinpath(dataDir, "data-glycan-gp120.csv"))
+    dfIGG = CSV.read(joinpath(dataDir, "data-luminex-igg.csv"))
+    dfL = CSV.read(joinpath(dataDir, "data-luminex.csv"))
+    dfMA = CSV.read(joinpath(dataDir, "meta-antigens.csv"))
+    dfMD = CSV.read(joinpath(dataDir, "meta-detections.csv"))
+    dfMG = CSV.read(joinpath(dataDir, "meta-glycans.csv"))
+    dfMS = CSV.read(joinpath(dataDir, "meta-subjects.csv"))
+    
+    df = meltdf(dfL, view = true)
+    newdfL = DataFrame(Rec = String[], Vir = String[], Sig = String[], Value = Float64[], Subject = Int64[])    
+for i in 1:88871
```

Comment:
Weird indenting going on with this for loop. Also, I realize you needed to handle this manually because of the data, but can you provide a comment about what this block of code does?